dictionary.txt

```diff
@@ -1053,3 +1053,117 @@ _rp	₎
 \latinCross	✝︎	
 \starAndCrescent	☪︎	
 \starOfDavid	✡︎	
+a•	ȧ	
```

Comment:
Maybe it would make sense to make this use only basic ascii characters similar to other shortcuts?

I think using something like `\dot_` for one dot and `\ddot_` for double would be more consistent.

Eg. `\dota -> ȧ`  `\ddota -> ä` etc.

Feel free to let me know what you think.

https://tex.stackexchange.com/questions/152951/how-to-write-two-dot-above-a-letter